.github/workflows/load-testing.yml

```diff
@@ -0,0 +1,24 @@
+name: load testing
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  load:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20.1.0
+
+      - name: Install Artillery
+        run: npm install -g artillery@latest
+
+      - name: Execute load tests
+        run: TARGET_URL=http://example.com artillery run artillery/*.yml
```

Comment:
It will be fantastic to create an [environment variable](https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows) for the node version and use it for all GitHub actions